agent/healthcheck/health_check.go

```diff
@@ -59,6 +59,9 @@ type HealthStatus struct {
 	ManagementServerDisconnectedTimestamp *time.Time `json:"managementServerDisconnectedTimestamp,omitempty"`
 	EnvoyReadinessState                   string     `json:"envoyReadinessState"`
 	InitialConfigUpdateStatus             string     `json:"initialConfigUpdateStatus,omitempty"`
+	HealthStatusFlipCount                 int        `json:"HealthStatusFlipCount,omitempty"`
```

Comment:
~~json:"HealthStatusFlipCount,omitempty"~~
json:"healthStatusFlipCount,omitempty" ??

Same for FlipTimestamps below

Reply:
This came from previous implementation. Will remove it